NotesApp/Scenes/List/ListViewController.swift

```diff
@@ -0,0 +1,168 @@
+        
+        button.addAction(UIAction(handler: { [weak self] _ in
+            self?.pushAddNoteViewController()
+        }), for: .touchUpInside)
+        
+        return button
+    }()
+    
+    // MARK: - Lifecycle
+    init(viewModel: ListViewModel) {
+        self.viewModel = viewModel
+        super.init(nibName: nil, bundle: nil)
+    }
+    
+    required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        setupUI()
+        setupBindings()
+    }
+    
+    // MARK: - UI Setup
+    private func setupUI() {
+        view.backgroundColor = UIColor(hex: "#252525")
+        
+        let views = [notesLabel, tableView, addNoteButton]
+        
+        views.forEach { subview in
+            view.addSubview(subview)
+            subview.translatesAutoresizingMaskIntoConstraints = false
+        }
+        
+        NSLayoutConstraint.activate([
+            notesLabel.leadingAnchor.constraint(equalTo: view.leadingAnchor, constant: 24),
+            notesLabel.topAnchor.constraint(equalTo: view.topAnchor, constant: 60),
+            
+            tableView.leadingAnchor.constraint(equalTo: view.leadingAnchor, constant: 24),
+            tableView.trailingAnchor.constraint(equalTo: view.trailingAnchor, constant: -24),
+            tableView.topAnchor.constraint(equalTo: notesLabel.bottomAnchor, constant: -10),
+            tableView.bottomAnchor.constraint(equalTo: view.bottomAnchor),
+            
+            addNoteButton.trailingAnchor.constraint(equalTo: view.trailingAnchor, constant: -25),
+            addNoteButton.bottomAnchor.constraint(equalTo: view.bottomAnchor, constant: -25)
+        ])
+    }
+    
+    // MARK: - Bindings
+    private func setupBindings() {
+        viewModel.onNotesUpdated = { [weak self] in
+            self?.tableView.reloadData()
+        }
+    }
+    
+    // MARK: - Button Actions
+    private func pushAddNoteViewController() {
+        let editorVM = EditorViewModel(note: nil)
+        let editorVC = EditorViewController(viewModel: editorVM)
+        editorVC.delegate = self
+        self.navigationController?.pushViewController(editorVC, animated: true)
+    }
+}
+
+// MARK: - Table View
+extension ListViewController: UITableViewDelegate, UITableViewDataSource {
+    func tableView(_ tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
+        return viewModel.noteCount()
+    }
+
+    func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
+        let cell = tableView.dequeueReusableCell(withIdentifier: NotesCellTableViewCell.reuseIdentifier, for: indexPath) as! NotesCellTableViewCell
+        let note = viewModel.note(at: indexPath.row)
+        cell.configure(with: note)
+        cell.backgroundColor = .clear
+        cell.selectionStyle = .none
```

Comment:
If this attribute won't be changed, it's better to write it inside NotesCellTableViewCell